### DISCUSSION TITLE
NeRF Discord Community

### DISCUSSION THREAD

**Discussion by franclucent** [NVlabs/instant-ngp#815] [General]
Hi Everyone! 

My name's Franc Lucent and I've been using Instant-NGP to capture memories of people. 

[I created a community for people using Instant-NGP and NeRFs to connect on Discord](https://discord.gg/atzsCcAXEh) and wanted to share it here!

**Comment by HungNgoCT** [comment]
Hi there.

I am using GPU RTX 2070 super with 8GB vRam. It can train small number of images (<~80). However, cannot apply for larger number of images, such as 100 images, because of out of GPU memory.

Also, I cannot increase resolution (such as 1080x1920) during training time for observation due to out of GPU memory too.

I understood that we can reduce "batch size", or chunk size... to save GPU memory. Any one can help me to understand the point in the code to reduce "batch size" like that? Also, Is there any way to save network, load and render out put later to save memory?

Highly appreciate for any one can help.
